### ISSUE TITLE
Error when running test on CPU

### ISSUE THREAD

**Issue by alessandrostranieri** [dicarlolab/CORnet#12]
The run script is failing when I don't specify the number of GPUs.

```
python run.py test --model S --data_path images --output_path features
```

Here's the stack trace:

```
Traceback (most recent call last):
  File "C:/Users/deral/University/Thesis/CORnet/run.py", line 350, in <module>
    fire.Fire(command=FIRE_FLAGS)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\fire\core.py", line 138, in Fire
    component_trace = _Fire(component, args, parsed_flag_args, context, name)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\fire\core.py", line 471, in _Fire
    target=component.__name__)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\fire\core.py", line 675, in _CallAndUpdateTrace
    component = fn(*varargs, **kwargs)
  File "C:/Users/deral/University/Thesis/CORnet/run.py", line 226, in test
    model(im)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\container.py", line 100, in forward
    input = module(input)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\container.py", line 100, in forward
    input = module(input)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\module.py", line 532, in __call__
    result = self.forward(*input, **kwargs)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\conv.py", line 345, in forward
    return self.conv2d_forward(input, self.weight)
  File "C:\Users\deral\Anaconda3\envs\PyThesis\lib\site-packages\torch\nn\modules\conv.py", line 342, in conv2d_forward
    self.padding, self.dilation, self.groups)
RuntimeError: Expected object of device type cuda but got device type cpu for argument #1 'self' in call to _thnn_conv2d_forward

Process finished with exit code 1
```

I try to debug the execution and I found that the checkpoint data is loaded correctly. At line https://github.com/dicarlolab/CORnet/blob/c04c1c027658a2184ade296feeb205b472bd46bd/cornet/__init__.py#L21 the weights in `state_dict` have their `device` correctly set to **cpu**. However, after `load_state_dict` is called, the weights in the model have their `device` set to **cuda:0**. Could the problem be there?

**Comment by alessandrostranieri** [comment]
I noticed that this happens on Windows, but not on Linux. Since I am not blocked, I will close the issue.

**Comment by qbilius** [comment]
OK, actually this may only fix #11. I don't have a Windows machine to investigate this further.
